apple/internal/apple_framework_import.bzl

```diff
@@ -353,6 +353,13 @@ target.
                 [apple_common.Objc, AppleFrameworkImportInfo],
             ],
         ),
+        "avoid_link": attr.bool(
```

Comment:
I like @kastiglione's suggestion: `bundle_only`. The doc can make it clear that it doesn't link.